### PR TITLE
fix: Api and PropTypes

### PR DESCRIPTION
**Pull request by AnnDv**

### CODE REVIEW

File: reactMovieApp/src/App.js [hunk 1]
```diff
@@ -4,9 +4,7 @@ import FetchMovies from './api/FetchMovies';
 
 function App() {
   return (
-    <>
-      <FetchMovies />
-    </>
+    <FetchMovies/>
   );
 }
```
Comment (nkv843) [inline reactMovieApp/src/App.js]: arrow func, inline return ?

File: reactMovieApp/src/api/FetchMovies.js [hunk 2]
```diff
@@ -1,26 +1,25 @@
+import { getData } from "./getData";
 import { CONFIG } from "../config";
-import React, { useEffect, useState } from "react";
 import MovieCard from "../components/MovieCard";
+import  React, { useEffect, useState } from "react";
 
 const API_URL = CONFIG.BASE_URL + CONFIG.POPULAR_MOVIE_URL + CONFIG.API_KEY;
```
Comment (nkv843) [inline reactMovieApp/src/api/FetchMovies.js:6]: is it fully correct to use react methods in API client file? 
maybe it's better to left here only fetching, like we did it in vanilla app?

File: reactMovieApp/src/components/MovieCard.js [hunk 3]
```diff
@@ -11,4 +10,8 @@ function MovieCard({movies}){
 	);
```
Comment (nkv843) [inline reactMovieApp/src/components/MovieCard.js:10]: arrow func, inline return 

ESLint  doesn't complain about the file type extension?

File: reactMovieApp/src/components/movieComponents/CustomModal.js [hunk 4]
```diff
@@ -5,8 +5,6 @@ import PropTypes from 'prop-types';
 
 export default function CustomModal({open, children, onClose}) {
   if(!open) return null;
-
-  console.log(typeof(onClose))
   
   return ReactDOM.createPortal(
     <>
```
Comment (nkv843) [inline reactMovieApp/src/components/movieComponents/CustomModal.js:10]: for what?

File: reactMovieApp/src/components/movieComponents/MovieDescription.js [hunk 5]
```diff
@@ -2,12 +2,10 @@ import './MovieDescription.css';
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const MovieDescription = (movie) => {
-    const info = movie.item;
-    
+const MovieDescription = ({overview}) => {
```
Comment (nkv843) [inline reactMovieApp/src/components/movieComponents/MovieDescription.js:5]: inline return

File: reactMovieApp/src/components/movieComponents/MovieImage.js [hunk 6]
```diff
@@ -5,17 +5,16 @@ import PropTypes from 'prop-types';
 
 const IMG_URL = CONFIG.IMG_URL;
 
-function MovieImage(movie) {
-
+function MovieImage({image}) {
```
Comment (nkv843) [inline reactMovieApp/src/components/movieComponents/MovieImage.js]: how come? why movie description is arrow func, but here you use functions expression

File: reactMovieApp/src/api/FetchMovies.js (hunk 2 above)
Comment (chitanumihai) [reply reactMovieApp/src/api/FetchMovies.js:6]: please move FetchMovies inside feature branch

File: reactMovieApp/src/api/getData.js [hunk 7]
```diff
@@ -0,0 +1,14 @@
+export async function getData(url) {
+    let data = [];
+    
+    try{
+        let response = await fetch(url)
```
Comment (chitanumihai) [inline reactMovieApp/src/api/getData.js]: please use const

File: reactMovieApp/src/api/getData.js [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+        .then(response => response.json())
+        .then(data => { 
+            return data.results;
+        });
+        data = response;
+    } catch(err) { alert(err) }
```
Comment (chitanumihai) [inline reactMovieApp/src/api/getData.js]: please return the error here, and dont mix dom related functions with api

**Pull request merged**